src/Build.OM.UnitTests/Construction/ProjectFormatting_Tests.cs

```diff
@@ -605,5 +603,86 @@ public void ChangeItemTypeWithXmlHeader()
 
             VerifyAssertLineByLine(expected, actual);
         }
+
+        [Fact]
+        public void VerifyUtf8WithoutBomTreatedAsUtf8()
+        {
+            string expected = ObjectModelHelpers.CleanupFileContents(@"<Project>
+</Project>");
+
+            Project project = new Project(NewProjectFileOptions.None);
+            StringWriter writer = new EncodingStringWriter(new UTF8Encoding(true));
```

Comment:
can you give the param name for the `true` param? I can't remember which direction it points.

(everywhere)